google-cloud-spanner/src/test/java/com/google/cloud/spanner/SpannerGaxRetryTest.java

```diff
@@ -303,18 +318,13 @@ public void singleUseExecuteStreamingSqlUnavailable() {
 
   @Test
   public void readWriteTransactionTimeout() {
-    expectedException.expect(SpannerMatchers.isSpannerException(ErrorCode.DEADLINE_EXCEEDED));
     mockSpanner.setBeginTransactionExecutionTime(ONE_SECOND);
-    TransactionRunner runner = clientWithTimeout.readWriteTransaction();
-    long updateCount =
-        runner.run(
-            new TransactionCallable<Long>() {
-              @Override
-              public Long run(TransactionContext transaction) throws Exception {
-                return transaction.executeUpdate(UPDATE_STATEMENT);
-              }
-            });
-    assertThat(updateCount, is(equalTo(UPDATE_COUNT)));
```

Comment:
Did @olavloite confirm that this is not needed?

Reply:
Yes, this part of the code will be unreachable. The database client that is used for these tests have configured `MinSessions=0`, which means that the error for `BeginTransaction` should bubble up to the user when a read/write transaction is requested.

Reply:
Perfect, thanks for confirming.